src/protocol.rs

```diff
@@ -502,6 +527,12 @@ impl<'a> fmt::Display for Protocol<'a> {
             Ip6(addr) => write!(f, "/ip6/{}", addr),
             P2pWebRtcDirect => f.write_str("/p2p-webrtc-direct"),
             P2pWebRtcStar => f.write_str("/p2p-webrtc-star"),
+            WebRTC => f.write_str("/webrtc"),
+            Certhash(hash) => write!(
+                f,
+                "/certhash/{}",
+                multibase::Base::Base58Btc.encode(hash.to_bytes())
```

Comment:
Note that the Golang implementation uses Base64 by default for the string representation. I guess it doesn't matter for machines, though it does for humans. Should we be consistent across implementations? I am not aware of a benefit of one over the other.

//CC @marten-seemann @ckousik

https://github.com/multiformats/go-multiaddr/pull/176/files#diff-38293d4bb2018b8660c43e7007e16a293752d0205fdc660558c419d6d3d199a6R392